### DISCUSSION TITLE
Stop scrollbar while input range move

### DISCUSSION THREAD

**Discussion by elenaramirez** [idiotWu/smooth-scrollbar#347] [Q&A] [answered]
Hello!

I'm using your awesome plugin (last version) in a site that includes input range elements on it. 
When i use those inputs in an ipad the page accidentally moves with the slide movement. I would like the scrolls keep paused while i move the range, i tried destroy() method but the page doesnt keep in the same position. 

Is there any option to momentarily stop the scrollbar movement? And restart it again when i mouseleave the input range?

Thank you so much.

**Comment by dolphin-wood** [comment, marked as answer]
Try:

```js
input.addEventListener('touchmove', (e) => {
  e.stopPropagation();
});
```

**Comment by elenaramirez** [comment]
So easy i want to cry. 

Thank you so much @idiotWu!! You rock!
